session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -88,7 +88,7 @@
 import java.util.stream.Collectors;
 
 @SuppressWarnings({"java:S107", "java:S1135"}) // need enough parameters, ignore todos
-public class Session {
+public class Session implements ISession {
```

Comment:
Methods need to be added `@Overried` annotation.

Reply:
done